src/main/java/com/patchworkmc/Patchwork.java

```diff
@@ -65,18 +68,20 @@
 	private Path clientJarSrg;
 	private IMappingProvider primaryMappings;
 	private List<IMappingProvider> devMappings;
-	private FieldDescriptorProvider fieldDescriptorProvider;
-	private NaiveRemapper naiveRemapper;
-	private AccessTransformerRemapper accessTransformerRemapper;
+	private PatchworkRemapper patchworkRemapper;
+	private Remapper accessTransformerRemapper;
+	private final IntermediaryHolder intermediaryHolder;
 	private boolean closed = false;
 
-	public Patchwork(Path inputDir, Path outputDir, Path dataDir, Path tempDir, IMappingProvider primaryMappings, List<IMappingProvider> devMappings) {
+	public Patchwork(Path inputDir, Path outputDir, Path dataDir, Path tempDir, Path bridgedMappings, List<IMappingProvider> devMappings) {
```

Comment:
I think we should continue passing the IMappingProvider here. Maybe have another parameter for the intermediary mappings per the comment below.